src/tracker/psmove_tracker.c

```diff
@@ -1545,7 +1545,7 @@ psmove_tracker_free(PSMoveTracker *tracker)
 int
 psmove_tracker_adapt_to_light(PSMoveTracker *tracker, float target_luminance)
 {
-    float minimum_exposure = 2051;
+    float minimum_exposure = 1; // Was 2051
```

Comment:
It may break non PS3EYEDriver targets. I don't know, I can't test. Where did those original numbers come from? They are not at all in the correct range for the PS3EYEDriver. Maybe they should not be hard-coded. See my comment below.